cfg/gc/run-excalibur.sh

```diff
@@ -29,18 +29,18 @@ cd -
 
 # Run analysis with config from worker node tmp dir
 echo "#### Starting Artus..."
-_TME_FILE='artus.tme'
+_TME_FILE='excalibur.tme'
 if [[ $(which time) ]]; then
 	# GNU time allows for extended format - use IEEE POSIX.2 as base
-	$(which time) -o ${_TME_FILE} -f 'real %e\nuser %U\nsys %S\nrss %M\nmajor_pfault %F\nmajor_pfault%R\nwaits %w' @WORKPATH@/artus @ARTUS_CONFIG@ || exit 1
+	$(which time) -o ${_TME_FILE} -f 'real %e\nuser %U\nsys %S\nrss %M\nmajor_pfault %F\nmajor_pfault%R\nwaits %w' excalibur @ARTUS_CONFIG@ || exit 1
```

Comment:
I'll just reply with "thank god I don't do CMSSW anymore". Carry on if you think it does what I don't think it does, then.